core/numbers/numbers_unprefixed.talon

```diff
@@ -0,0 +1,3 @@
+not tag: user.disable_unprefixed_numbers
```

Comment:
I believe we still want
 and not tag: user.mouse_grid_showing
here

Reply:
I don't think so because mouse_grid is updated to set the tag, which would disable numbers_unprefixed.talon. As per #943 the idea of doing it the way I did was that the core/numbers code doesn't need to be aware of the existence of mouse grid which I think is preferable. This way mouse grid itself can just toggle the functionality off, so its own functionality won't conflict. 

The downside of this is that it's a double negative tag. To me having core/numbers not having to special case for some plugin is worth the slightly confusing double negative tag, but if others disagree then can change it.